tools/findMissingLocales.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs')
```

Comment:
I think locales.js would be a better name.

Reply:
Hmm.. I don't think so... that doesn't tell you what the file actually does...